### DISCUSSION TITLE
Token set cross-references

### DISCUSSION THREAD

**Discussion by derekluciani** [six7/figma-tokens#229] [Q&A] [answered]
Is cross-token-set referencing supported? 

I want to create a Line Height scale that references the Font Size tokens I've created.

I get a null result (not working) whenever I try to create the following: 

![line-height-scale-reference](https://user-images.githubusercontent.com/16504121/131394152-ee9cfd6b-c52f-45b5-849e-2a6c7c2bb2e4.png)

**Comment by six7** [comment, marked as answer]
Yes! You'd need to check the token sets you want activated, the plugin will look in these for tokens when resolving aliases.

**Reply by derekluciani** [reply]
Thank you, Jan.
